DigitalLearningSolutions.Data/Models/User/CentreAnswersData.cs

```diff
@@ -30,5 +33,19 @@ public class CentreAnswersData
         public string? Answer4 { get; set; }
         public string? Answer5 { get; set; }
         public string? Answer6 { get; set; }
+
+        public string? GetAnswerForRegistrationPromptNumber(int promptNumber)

```

Comment:
Using the RegistrationField enumeration would be a bit safer than adding knowlege here about which field has which id here (sad that you can't use a switch statement though).

Reply:
Not quite sure what you mean about "adding knowledge", the fields are intrinsically linked to answers through the Id. I've changed the parameter around so it should be safer. If this isn't what you had in mind, let me know.

Reply:
This is fine, sorry.

If your definition of RegistrationField.Id is "the answer number in the database" then yeah, it is intrinsically linked. That's not explicit in the codebase (might have been nicer to make it so by having it as an extra field "int? answerNumber" on RegistrationField, but I don't think it's worth it).

I think this is a big improvement over having an `int` param, because the caller no longer has to worry about what a "registration prompt number" is - it could have been the straight id, or the linkedToFieldId.

Reply:
Nice, I'll merge this then.

But for reference, I thought the whole point of the RegistrationField enumeration was so that we had an explicit link between the codebase and answer number. The ID being the answer field number, with 7 reserved for the special case of Job Group. At least when the refactor to include this RegistrationField was implemented, the ID was used in most places where prompt numbers came about, so I thought that was the point.

Reply:
Yeah, and that is how it's used. It's just not explicit from the name. It's not a big logical jump to assume that that's the connection between the RegistrationField enumeration and the Answer numbers so I don't think we need to do anything.